dynamic-plugins.default.yaml

```diff
@@ -426,6 +426,33 @@ plugins:
                     - isKind: resource
                     - isType: kubernetes-cluster
 
+  # Group: Orchestrator
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator
```

Comment:
the path inside the image is `./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator-backend-dynamic`